app/tests/spec/models/auth_brokers/oauth-redirect.js

```diff
@@ -230,7 +509,7 @@ define(function (require, exports, module) {
         return broker.afterCompleteResetPassword(account)
```

Comment:
Something about this whole test no longer makes sense to me, the test description doesn't match anything in the code. @vbudhram do you remember what this is meant to test?

Reply:
I have made the required changes @shane-tomlinson. 
Even I don't know what's this test is for. Although the assertions in this are very much similar to the assertions in `ignores account verified if verification method and reason set`.